src/hotspot/cpu/ppc/stubGenerator_ppc.cpp

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 1997, 2019, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2012, 2019 SAP SE. All rights reserved.
+ * Copyright (c) 1997, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2020, SAP SE. All rights reserved.
```

Comment:
No comma before SAP SE, please! (See https://bugs.openjdk.java.net/browse/JDK-8252837)

Reply:
Interesting.  I was trying to make it consistent with the Oracle copyright.  Ok, will fix.